src/Benchmarks.jl

```diff
@@ -1,10 +1,24 @@
 module Benchmarks
-    export @benchmark
+    export @benchmark,
+           BenchmarkResults,
```

Comment:
I would prefer not to export so many names. I'm trying to do my part to remove the culture of `using` pulling in a ton of names.